Jenkinsfile.yml

```diff
@@ -61,75 +61,62 @@ node('ccode'){
-//          cp sdo-test/testFiles/keys/ecdsa256privkey.dat sdo-test/binaries/client-sdk/x86_ecdsa256_sct_bin/data/
-//          tar -xvzf client-sdk/x86_ecdsa384_c_sct_bin.tar.gz -C sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin/ --strip=1
-//          cp sdo-test/testFiles/keys/ecdsa384privkey.dat sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin/data/
-//          cp -r sdo-test/binaries/client-sdk sdo-test
-//        '''
-//        print "Create tomcat containers running manufacturer & db"
-//        sh '''
-//          cd sdo-test/binaries/supply-chain-tools/docker_manufacturer/
-//          docker-compose up -d --build
-//          docker logs manufacturer-mariadb
-//          docker logs manufacturer
-//        '''
-//        print "Client-SDK Device smoke Tests with PRI-RV, SCT-Manufacturer"
-//        sh '''
-//          cd sdo-test
-//          mvn clean verify -Dgroups=CdeviceTests
-//        '''
-//        print "Client-SDKDevice smoke Tests with Onprem-RV, SCT-Manufacturer"
-//        sh '''
-//         cd sdo-test
-//          mvn clean verify -Dgroups=CdeviceOnPremTests
-//        '''
-//        }
-//    } finally {
-//        print "stop tomcat containers running manufacturer & db"
-//        sh '''
-//          docker logs manufacturer-mariadb > mariadbLog.txt
-//          docker logs manufacturer > manufacturerLog.txt
-//          docker stop $(docker ps -a -q)
-//          docker rm -v $(docker ps -a -q) -f
-//          docker system prune -a -f
-//          docker images -a
-//          docker volume ls
-//          docker ps -a
-//        '''
-//        print "Archive the smoke test logs "
-//        sh 'mkdir SmokeTest_Log'
-//        sh '''
-//          cp -r manufacturerLog.txt SmokeTest_Log/
-//         cp -r mariadbLog.txt SmokeTest_Log/
-//          cp -r sdo-test/logs/*_log.txt SmokeTest_Log/
-//          tar -zcvf SmokeTest_Log.tar.gz SmokeTest_Log
-//        '''
-//        archiveArtifacts artifacts: 'SmokeTest_Log.tar.gz', fingerprint: true, allowEmptyArchive: false
-//    }
-    //Clean workspace after build is successfull
+
+  try
+  {
+    stage('Run Smoketest')
+    {
+      // Checking out the smoke test code
+      checkout([$class: 'GitSCM', branches: [[name: '*/master']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'sdo-test']], userRemoteConfigs: [[credentialsId: 'sdo-automationgithubtoken', url: 'https://github.com/secure-device-onboard-ci/sdo-test']]])
+
+      // Creating the required directories
+      sh '''
+        mkdir -p sdo-test/binaries/pri-fidoiot
+        mkdir -p sdo-test/binaries/client-sdk-fidoiot
+      '''
+
+      // Copying the contents to the folder
+      print "Copying PRI-FIDOIOT and CSDK-FIDO artifacts"
+      copyArtifacts filter: 'demo.tar.gz', fingerprintArtifacts: true, projectName: 'pri-fidoiot/master', selector: lastSuccessful(), target: 'sdo-test/binaries/pri-fidoiot/'
+      print "Extracting PRI-FIDOIOT and Client-SDK-FIDO artifacts"
```

Comment:
Use uniform terminologies (other than repo names) while referring to PRI and Client-SDK.

PRI-FIDOIOT => `FDO PRI` or `PRI`
Client-SDK-FIDOIOT => `FDO Client-SDK` or `Client-SDK`